packages/zowe-explorer/src/tools/ZoweTerminal.ts

```diff
@@ -0,0 +1,175 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { imperative } from "@zowe/zowe-explorer-api";
+import * as vscode from "vscode";
+
+export class ZoweTerminal implements vscode.Pseudoterminal {
+    public static readonly mTermX = ">";
+    public static readonly Keys = {
+        EMPTY_LINE: `${this.mTermX} `,
+        CLEAR_ALL: "\x1b[2J\x1b[3J\x1b[;H",
+        CLEAR_LINE: `\x1b[2K\r`,
+        CTRL_C: "\x03",
+        DEL: "\x1b[P",
+        ENTER: "\r",
+        NEW_LINE: "\r\n",
+        UP: "\x1b[A",
+        DOWN: "\x1b[B",
+        RIGHT: "\x1b[C",
+        LEFT: "\x1b[D",
+        BACKSPACE: "\x7f",
+    };
```

Comment:
Thanks for working on these terminal functionalities 😋  I think folks will really enjoy how close it feels to a real terminal session.